src/main/java/ui/export/PDFCompiler.java

```diff
@@ -23,7 +25,8 @@ private PDFCompiler() {
 	}
 
 	public static Optional<String> compileToPDF(Global global, Month month, File targetFile) {
-		try (InputStream templateStream = Object.class.getResourceAsStream("/pdf/template.pdf")) {
+		// Object.class.getResourceAsStream cannot find the pdf, at least not on Win11
+		try (InputStream templateStream = Thread.currentThread().getContextClassLoader().getResourceAsStream("pdf/template.pdf")) {
```

Comment:
just create a constant like this:

`private static final Object LOADER = new Object(){};`

Then, you can use `LOADER.getClass().getResourceAsStream("/pdf/template.pdf")`

Reply:
![image](https://github.com/user-attachments/assets/3a2c5bd3-8215-457a-826a-fd3ec2ad6eef)

This also doesn't work for some reason I haven't been able to figure out. It did work before, but it can't find the PDF now when using Object.class.getResourceAsStream in any way, shape or form.

Reply:
My code is a little different from Object.class.getClass :D The brackets are important :D

Reply:
I copied your code 1:1, it still didn't work ¯\_(ツ)_/¯

I don't exactly know what the issue is, since nothing really changed and it worked previously, but the new version works on both OS I tested. I also tested lots of different variations of writing the path, and also adding the PDF directory directly as resource root in maven and just look for "template.pdf".

Reply:
Nevermind, I got it to work